### DISCUSSION TITLE
job search project

### DISCUSSION THREAD

**Discussion by eerison** [shield-wall/lunarvim-docker#6] [Ideas]
Let's use this thread to bring ideas for the next open source project

**Comment by Willian1661** [comment]
about the name i was thinking to name it: DoneDeal/ Hiring and Seek.
one of these as example.

**Reply by eerison** [reply]
The name can be in Portuguese:)

**Reply by eerison** [reply]
Or a mix

**Reply by eerison** [reply]
Can also be in English but it's not mandatory,  the first version will be in Portuguese

**Comment by Willian1661** [comment]
for the layout what do you think be something between glassdor and indeed?
![Screenshot from 2023-03-03 19-46-33](https://user-images.githubusercontent.com/106893704/222846724-5dfe7358-f0bc-4360-9680-d988e1e283b4.png)
![Screenshot from 2023-03-03 19-46-51](https://user-images.githubusercontent.com/106893704/222846753-73c02686-3bd1-4fdb-a50f-568af464e02c.png)

or maybe should be something different from them, like more original 🤔?

**Reply by eerison** [reply]
> for the layout what do you think be something between glassdor and indeed? ![Screenshot from 2023-03-03 19-46-33](https://user-images.githubusercontent.com/106893704/222846724-5dfe7358-f0bc-4360-9680-d988e1e283b4.png) ![Screenshot from 2023-03-03 19-46-51](https://user-images.githubusercontent.com/106893704/222846753-73c02686-3bd1-4fdb-a50f-568af464e02c.png)
> 
> or maybe should be something different from them, like more original 🤔?

More original

**Reply by eerison** [reply]
We can do something more simple for now, just with is more important

**Reply by Willian1661** [reply]
ok, but how can i show you a layout? i mean i draw something and send you here
or it's more like a discussion about it's gonna have or not or how things will be placed

**Reply by eerison** [reply]
maybe a behance layout?

**Comment by eerison** [comment]
job list: https://www.behance.net/gallery/165041551/Job-portal-site?tracking_source=search_projects%7Cjob+site
job detail: https://www.behance.net/gallery/127008283/Job-Board-X-Job-Board-Webflow-Template?tracking_source=search_projects%7Cjob+site

**Comment by eerison** [comment]
<img width="1258" alt="Screenshot 2023-03-10 at 18 04 28" src="https://user-images.githubusercontent.com/6358755/224378007-fabf0c67-96b6-4979-9ee6-5fc5271dbbeb.png">
